docs/source/topics/cache.rst

```diff
@@ -32,6 +32,25 @@ In addition, the cache is culled based on ``CULL_FREQUENCY`` when  ``add()``
 or ``set()`` is called, if ``MAX_ENTRIES`` is exceeded. See
 :ref:`django:cache_arguments` for an explanation of these two options.
 
+Cache entries include a HMAC signature to ensure data integrity by default.
+You can disable this by setting ``ENABLE_SIGNING`` to ``False``.  
+Signatures can also include an optional key and salt parameter by setting 
```

Comment:
What is the use case for custom key and salt? Probably it should be explained that SECRET_KEY is the default.

Reply:
As suggested by another reviewer, I removed the SALT option since it wasn't providing what I expected it to. I also added the documentation of `SECRET_KEY` as requested. 
I don't have a valid use case for changing the key, but I feel like we would be limiting developers by arbitrarily removing the option when it doesn't really cost anything to keep it. Do you think it's fine as is, or should we brainstorm some reasons to keep/remove it?